package.json

```diff
@@ -6,7 +6,9 @@
     "check-types": "tsc --noEmit",
     "start": "react-native start",
     "test:unit": "jest --passWithNoTests",
-    "format": "prettier --write --list-different './src/**/*.{js,ts,tsx}'",
+    "format": "prettier --write --list-different './src/**/*.{js,ts,tsx}' && yarn clang-format && yarn lint-java",
+    "lint-java": "node ./scripts/lint-java.js",
+    "clang-format": "find ios/ -iname *.h -o -iname *.m -o -iname *.cpp | xargs clang-format -i",
```

Comment:
For this to work, you have to have `clang-format` installed on your machine. We should definitely add a [clang-format](https://github.com/angular/clang-format#readme) package as a dev-dependency to the repo. That way we avoid a situation where a (to be) contributor can't even commit the code without an additional setup.